src/LineUI/LineSet.tsx

```diff
@@ -203,12 +211,27 @@ const LineSet : React.FC<ILineSetProps> = ({ getCTM, boundaries, unit, size, lin
     />
   );});
 
+  const AreaLabel = () => {
```

Comment:
can we move this component out of the LineSet and just pass props for the points and areaName?

Reply:
Done